app/components/entryReview/EntryComments/CommentForm/index.tsx

```diff
@@ -121,21 +121,26 @@ function CommentForm(props: Props) {
             >
                 <TextArea
                     name="text"
+                    className={styles.commentArea}
                     rows={3}
                     value={value.text}
                     onChange={setFieldValue}
                     error={error?.text}
                     autoFocus
+                    // TODO: create a new variant for input container
+                    // with removed border and filled background
                 />
                 <ProjectMemberMultiSelectInput
                     name="mentionedUsers"
+                    inputSectionClassName={styles.selectInput}
                     label="Assignees"
                     value={value.mentionedUsers}
                     projectId={projectId}
                     onChange={setFieldValue}
                     options={members}
                     onOptionsChange={setMembers}
                     error={getErrorString(error?.mentionedUsers)}
+                    variant="general"
```

Comment:
What does this variant do?

Reply:
@subinasr ?

Reply:
It WAS for the styling purpose, but it's removed now.